tests/test_lua_details.cc

```diff
@@ -22,10 +22,21 @@
 
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
+#include <limits>
 #include <gul14/catch.h>
-
 #include "../src/lua_details.h"
 
+using namespace std::chrono;
 using namespace std::literals;
 using namespace task;
 
+TEST_CASE("get_ms_since_epoch()", "[lua_details]")
+{
+    const auto now = Clock::now();
+
+    REQUIRE(get_ms_since_epoch(TimePoint{}, 100ms) == 100);
+    REQUIRE(get_ms_since_epoch(now, 100ms)
+            == round<milliseconds>((now + 100ms).time_since_epoch()).count());
+    REQUIRE(get_ms_since_epoch(now, milliseconds::max())
+            == std::numeric_limits<long long>::max());
```

Comment:
I'm not sure about this test?

Reply:
`long long get_ms_since_epoch()` says it returns "the maximum representable time point ... in case of overflow", "representable" meaning "in terms of `long long`.

Hmmm... strictly speaking this test works only if the representation of `TimePoint` and `milliseconds` is `long long`. Otherwise there is no overflow from which the function needs to protect us.

Testing this correctly and portably would involve some template magic, I guess. I'll simply remove it for now. This was not the point of the PR after all.

Reply:
Apparently I was a bit too tired yesterday. :)
Now I made the test conditional with:
```
if constexpr (milliseconds::max().count() >= std::numeric_limits<long long>::max())
```
That should make it safe against weird `chrono` implementations.

Reply:
> Not sure why the code in `src/lua_details` uses `long long` and not `auto` or `milliseconds.rep`?

Because the value needs to be stuffed into the Lua registry, and that needs `long long`.

Reply:
Ah, THAT is the reason I proposed to use `LUA_INTEGER` instead of `long long`, that would make that explicitly clear.

As I pointed out elsewhere... We should have used `LUA_INTEGER` and the other types and/or introduced more c++ looking aliases like `using lua_integer = LUA_INTEGER` and use that.

Your reasoning was ... we do not have suffixes for that (i.e. `0LL`), but what is wrong with `lua_integer{ 0 }`...

That would clear all that stuff up, and make the need (for `long long`) very obvious.

Reply:
+1 for `LuaInteger`, if someone comes up with a PR. 😁